docs/installation.md

```diff
@@ -21,9 +20,16 @@ This document ends with these appendices:
 This installation guide assumes that the following softwares are already installed on the target system:
 
 * one of CentOS, Debian, FreeBSD or Ubuntu
-* sudo
+* sudo (only for installation and other administrative tasks)
+* cpanm (only for installation)
 * curl (only for post-installation sanity check)
 
+It also assumes that you've chosen (though not necessarily installed) one of the following database engines:
+
+* MySQL
+* PostgreSQL
+* SQLite
+
 

```

Comment:
The outline of the document differs slightly from your listing, so I can't simply add it without reorganizing the text.

What I believe you're asking for is a more concise explanation of the procedure. To that end I've reworked the beginning of the document. I had trouble describing the "Prerequisites" section. Turns out this section was complex so I broke it into two simple sections.